specs/WinRT/WinRTAPIContracts.md

```diff
@@ -22,18 +22,22 @@ WinRT APIs.
    1. One contract per feature/component/sub-system
```

Comment:
Why? What's the motivation/scenario?

Reply:
Past experience with `Windows.Foundation.UniversalApiContract` being too monolithic

Back when contracts were created the competing options were contracts per feature/component/subsystem vs 1 contract per Windows release. We chose the latter, 'cause, reasons, but WinAppSDK != Windows so our needs and policies differ. We discussed this in detail recently and agreed the per feature/component/subsystem modularity is a better fit for our needs than Windows' 1 per release.